go-controller/pkg/cni/cniserver_windows.go

```diff
@@ -0,0 +1,35 @@
+// +build windows
+
+package cni
+
+import (
+	"fmt"
+	"net"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	utilwait "k8s.io/apimachinery/pkg/util/wait"
+)
+
+// Start the Server's local HTTP server on a root-owned Unix domain socket.
+// requestFunc will be called to handle pod setup/teardown operations on each
+// request to the Server's HTTP server, and should return a PodResult
+// when the operation has completed.
+func (s *Server) Start(requestFunc cniRequestFunc) error {
+	if requestFunc == nil {
+		return fmt.Errorf("no pod request handler")
+	}
+	s.requestFunc = requestFunc
+
+	l, err := net.Listen("tcp", serverTCPAddress)
```

Comment:
So this is not protected, right?  eg it's a TODO to make this HTTPS or sometihng, right?

Reply:
TODO to use named pipes instead of tcp, that offers more security features